### DISCUSSION TITLE
How to apply font size ,font family, bold, italic etc only for the selected cell in table

### DISCUSSION THREAD

**Discussion by Albinbritto** [JiHong88/SunEditor#1353] [Q&A]
I have one requirement,I need to apply font size ,font family, bold, italic etc only for the selected cell in table.whether it is possible now?

**Comment by JiHong88** [comment]
This can be added in the 3.0.0 development version.
There is currently no feature.

**Reply by Albinbritto** [reply]
Great. when we can expect 3.0.0 version stable release?

**Reply by conorcrotch-git** [reply]
Been two years now, any update on when v3 is coming?
